trendly/src/app/country-input/country-input.component.html

```diff
@@ -0,0 +1,15 @@
+
+<mat-form-field class="form-field">
+  <input (change)="emitCountry()" type="text"
+           placeholder="Pick Country"
```

Comment:
Try to be consistent with letter casing in user visible UI
```suggestion
           placeholder="Pick a country"
```